modules/update-upgrading-cli.adoc

```diff
@@ -76,18 +76,18 @@ VERSION IMAGE
 4.10.13 quay.io/openshift-release-dev/ocp-release@sha256:4f516616baed3cf84585e753359f7ef2153ae139c2e80e0191902fbd073c4143
 ----
 
-. Based on your organization requirements, set the upgrade channel to `stable-4.10`, `fast-4.10`, or `eus-4.10`:
+. Based on your organization requirements, set the upgrade channel to \`stable-{product-version}`, \`fast-{product-version}`, or \`eus-{product-version}`:
 +
 [source,terminal]
 ----
 $ oc adm upgrade channel <channel>
 ----
 +
-For example, to set the channel to `stable-4.10`:
+For example, to set the channel to \`stable-{product-version}`:
```

Comment:
I think these will be fine without the `\` escape character. Currently in the preview build, they're showing up as literal backticks.